src/signals/incident/components/form/MapSelectors/Caterpillar/CaterpillarLayer/CaterpillarLayer.tsx

```diff
@@ -97,25 +106,42 @@ export const CaterpillarLayer: FC = () => {
 
   const reportedFeatureType = meta.featureTypes.find(
     ({ typeValue }) => typeValue === 'reported'
-  ) as FeatureType
-  const reportedFeatures = features.filter((feature) =>
-    Boolean(
-      reportedFeatureType?.isReportedField &&
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        feature.properties![reportedFeatureType.isReportedField] ===
-          reportedFeatureType.isReportedValue
-    )
+  ) as ReportedFeatureType
+  const checkedFeatureType = meta.featureTypes.find(
+    ({ typeValue }) => typeValue === 'checked'
+  ) as CheckedFeatureType
+
+  const statusFeatures = features.filter(
+    (feature) =>
+      Boolean(
+        reportedFeatureType.isReportedField &&
```

Comment:
`reportedFeatureType` can be `undefined`

Reply:
✅